lib/tpl/dokuwiki/tpl_header.php

```diff
@@ -19,13 +19,18 @@
 
         <h1 class="logo"><?php
             // get logo either out of the template images folder or data/media folder
-            $logoSize = array();
-            $logo = tpl_getMediaFile(array(':wiki:logo.png', ':logo.png', 'images/logo.png'), false, $logoSize);
+            $logoSize = [];
+            $logo = tpl_getMediaFile([
+                ':wiki:logo.svg', ':logo.svg',
+                ':wiki:logo.png', ':logo.png',
+                'images/logo.svg', 'images/logo.png'
+            ]);
 
             // display logo and wiki title in a link to the home page
             tpl_link(
                 wl(),
-                '<img src="'.$logo.'" '.$logoSize[3].' alt="" /> <span>'.$conf['title'].'</span>',
+                '<img src="' . $logo . '" ' . ($logoSize ? $logoSize[3] : '') . ' alt="" />' .
```

Comment:
`$logoSize` is always an empty array (line 22), so this check always results in `''`.

Reply:
you should restore passing it as third arg to `tpl_getMediaFile`?

Reply:
you're right. fixed.